stan-dev/stan
# Bugfix/2673 docs makefile

#### Submission Checklist


#### Summary
fixed path specifications in makefile `make/manual`

#### Intended Effect
allows cmdstan's task `make/manual` to succeed.

#### How to Verify
run cmdstan task `make manual` - verify that versioned pdfs for Stan reference manual and functions manual are copied over to directory cmdstan/doc

#### Side Effects
none

#### Documentation
N/A

#### Copyright and Licensing

Please list the copyright holder for the work you are submitting (this will be you or your assignee, such as a university or company): Columbia University



By submitting this pull request, the copyright holder is agreeing to license the submitted work under the following licenses:
- Code: BSD 3-clause (https://opensource.org/licenses/BSD-3-Clause)
- Documentation: CC-BY 4.0 (https://creativecommons.org/licenses/by/4.0/)
